extensions/wikia/UserLogin/js/FacebookFormConnectUser.js

```diff
@@ -0,0 +1,29 @@
+/* global UserBaseAjaxForm */
+
+(function () {
+	'use strict';
+
+	var FacebookFormConnectUser = function (el, options) {
+		UserBaseAjaxForm.call(this, el, options);
+	};
+
+	FacebookFormConnectUser.prototype = Object.create(UserBaseAjaxForm.prototype);
+
+		// Handles existing user login via modal
```

Comment:
This comment should probably be indented back one tab to be aligned with ```FacebookFormConnectUser``` below it.